package.json

```diff
@@ -18,12 +18,12 @@
     "docs": "typedoc",
     "lint": "eslint --ignore-path .gitignore \"src/**/*.ts\" --fix",
     "prepublishOnly": "npm run clean:all && npm i && npm run build && npm t",
-    "start:e2e": "ws -d tests/e2e/app  -p 8443 --cors.origin null --cors.credentials --cert ./example.com+5.pem --key ./example.com+5-key.pem",
+    "start:e2e": "ws -d tests/e2e/app  -p 8443 --cors.origin null --cors.credentials --https",
     "start:server": "node tests/e2e/server/index.mjs",
     "start:server:live": "LIVE=true node tests/e2e/server/index.mjs",
     "start:samples": "ws -d samples -p 8443 --cors.origin null --cors.credentials --cert example.com+5.pem --key example.com+5-key.pem",
     "test": "npm run test:unit && npm run test:integration && npm run test:e2e",
-    "test:coverage": "jest --testMatch=\"<rootDir>/src/**/*.test.ts\" --config=./tests/jest.coverage.config.js --env=\"<rootDir>/tests/jest.env.config.js\"",
+    "test:coverage": "jest --testMatch=\"<rootDir>/src/**/*.test.ts\" --config=./tests/jest.basic.config.js --coverage=true",
```

Comment:
I recently made modifications to this due to an unwanted side-effect. This would change it back to creating the side-effect. The issue was sharing the basic config was creating coverage reports for each test run, rather than just when "coverage" was run. So, our directories were getting polluted with hundreds of coverage reports.

With the update of Jest, did they fix that issue? I don't mind leaving it like this if you can ensure the same issue doesn't return.

Reply:
Good catch, I don't think I intended for this to carry over

Reply:
I can verify if they fixed the issue